backend/remote/remote.go

```diff
@@ -151,7 +151,7 @@ func (r *Remote) GetRevisionCounter() (int64, error) {
 	if err != nil {
 		return 0, err
 	}
-	if replica.State != "open" && replica.State != "dirty" {
+	if replica.State != "open" && replica.State != "dirty" && replica.State != "rebuilding" {
```

Comment:
Please state the necessity of the above and reverify all the scenarios where the replica goes down in rebuilding state.

Reply:
Thanks for pointing this out but this check is unnecessary now as we are persisting the revision count.